src/Service.Host/client/src/components/__tests__/parts/partReducer.test.js

```diff
@@ -73,7 +73,13 @@ describe('partReducer tests', () => {
 
     it('should handle simple field changes', () => {
         expect(
-            partReducer({}, { type: 'fieldChange', fieldName: 'partNumber', payload: 'PART' })
+            partReducer({}, { type: 'fieldChange', fieldName: 'partNumber', payload: ' PART' })
```

Comment:
should this just be 'PART' to demonstrate correct state

Reply:
ah yeah, I must've changed this one by accident and forgot to undo!